text_summary.py

```diff
@@ -0,0 +1,30 @@
+import torch
+from transformers import T5Tokenizer, T5ForConditionalGeneration
+
+model = T5ForConditionalGeneration.from_pretrained('t5-small') # use pretrained model
+tokenizer = T5Tokenizer.from_pretrained('t5-small') # use pretrained tokenizer
+device = torch.device('cpu')
+
+
+def text_summary(text):
+    '''
+    Abstractive text summarization
+    :param text: body of text to be summarized
+    :return: summarized version of original text
+    '''
+    # preprocessing
+    clean_text = text.strip().replace("\n","")
+    t5_text = "summarize: " + clean_text # define task
+
+    #tokenizer
+    tokenized_text = tokenizer.encode(t5_text, return_tensors="pt").to(device)
+    # summmarize
+    summary_ids = model.generate(tokenized_text,
```

Comment:
Nit: what does `summary_ids` stand for? If you mean "index" I'd suggest `summary_idx` instead